components/email/src/methods/participants.ts

```diff
@@ -0,0 +1,65 @@
+import type { Message, Participant } from "@commons/types/Nylas";
+
+export function isFromMe(myEmail: string, message: Message): boolean {
+  if (!myEmail) {
+    return false;
+  }
+
+  return message.from
+    .map((f) => {
+      return f.email.toLowerCase();
+    })
+    .includes(myEmail.toLowerCase());
+}
+
+export function participantsWithoutMe(
+  myEmail: string,
+  message: Message,
+): Participant[] {
+  const allParticipants = [
+    ...message.from,
+    ...message.to,
+    ...message.cc,
+    ...message.bcc,
+  ];
+  return allParticipants.filter((e) => e.email != myEmail);
+}
+
+export function wasCCed(myEmail: string, message: Message): boolean {
+  return message.cc.some((e) => e.email === myEmail);
+}
```

Comment:
Since this and my suggestion for `isFromMe()` are pretty much the same, why not make it a generic function where you can pass the field:
```
export function includesMyEmail(myEmail: string, message: Message, field: string): boolean {
  return message[field].some((e) => e.email.toLowerCase() === myEmail.toLowerCase());
```